packages/shared-state/files/etc/config/shared-state

```diff
@@ -0,0 +1,2 @@
+config shared-state 'options'
+	#list candidates '10.13.0.2'
```

Comment:
I think complete URLs should be provided instead of IP, it is more comform with the command line option and more flexible. If the config is an URL it is much easier to put a cgi script into whatever web-service at whatever path of which the use have less degree of control, instead with the IP the web-service must behave more stringently like a shared-state daemon.

Reply:
You're right, I already made the change.